pkg/arvo/sur/metadata-store.hoon

```diff
@@ -0,0 +1,23 @@
+|%
++$  group-path  path
++$  app-name  @tas
++$  app-path  path
```

Comment:
These feel like they might want an encapsulating `+$  resource  [=app-name =app-path]`.

Reply:
I like that.